### DISCUSSION TITLE
Instagram bot

### DISCUSSION THREAD

**Discussion by kevinkurian7** [prathimacode-hub/Awesome_Python_Scripts#15] [Q&A]
I have made a program which can log into your instagram with the credentials provided and search for post on instagram with the help of tags you have given in a text file.the bot then likes and comments on the posts  .

this programs makes use of the selenium library to scrape the web.

**Comment by prathimacode-hub** [comment]
This instagram bot clearly goes into WebScrapingScripts folder. What's the other program you were talking about, something related to maps? Also make sure u follow contributing guidelines. Earlier pull request missed the guidelines. Kindly correct it. And is that maps program automated fully? @KevinKurian7

**Comment by prathimacode-hub** [comment]
The format of adding your program into this repository should look like :

prathimacode-hub/Awesome_Python_Scripts/WebScrapingScripts/InstagramBot/Instagram_Bot.py

Also make sure you add a README file with short description with requirements needed. Screenshot inclusion is must.

Looking forward to merge your PR soon. Keep doing the great work. 👍 @KevinKurian7
